### PR TITLE
test: cypress - Added tests for Range Slider

### PR DESCRIPTION
**Pull request by NandanAnantharamu**
Range slider tests added, based on the tests that were not covered in below link
https://github.com/appsmithorg/TestSmith/issues/2066
https://github.com/appsmithorg/TestSmith/issues/2064
https://github.com/appsmithorg/TestSmith/issues/2061

### CODE REVIEW

**Review by sharanya-appsmith — APPROVED** [review]

**Comment by Aishwarya-U-R** [comment]
@NandanAnantharamu - For so many runs, please use limited run, only for this spec. Complete ok-to-test is not required.

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/Sliders/RangeSlider_spec.ts [hunk 1]
```diff
@@ -1,181 +1,423 @@
...
+    );
+
+    // Updating to allowed value
+    propPane.UpdatePropertyFieldValue("Min. range", "5");
+    agHelper.AssertElementAbsence(locators._evaluatedErrorMessage);
   });
 
-  it("5. Validates Default start value", () => {
-    _.propPane.UpdatePropertyFieldValue("Default start value", "-100");
+  it("6. Validate accepted and unaccepted 'Default start' value", () => {
```
Comment (sharanya-appsmith) [inline app/client/cypress/e2e/Regression/ClientSide/Widgets/Sliders/RangeSlider_spec.ts:322]: can default start and default end be combined?

**Review by sharanya-appsmith — APPROVED** [review]

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/Sliders/RangeSlider_spec.ts (hunk 1 above)
Comment (NandanAnantharamu) [reply app/client/cypress/e2e/Regression/ClientSide/Widgets/Sliders/RangeSlider_spec.ts:322]: done

**Comment by NandanAnantharamu** [comment]
closing this PR as i faced some bad merge issues , taken care in https://github.com/appsmithorg/appsmith/pull/26263

**Pull request closed without merging**